src/main/js/components/dropdowns/utils.js

```diff
@@ -19,15 +19,26 @@ function generateDropdown(element, callback, immediate) {
   tippy(
     element,
     Object.assign({}, Templates.dropdown(), {
-      hideOnClick: element.dataset["hideOnClick"] !== "false",
```

Comment:
Used in https://github.com/jenkinsci/jenkins/blob/e34bdcfdc0665c064dd825cf0d33a949bfd23c44/src/main/js/components/dropdowns/autocomplete.js#L96 originally introduced in https://github.com/jenkinsci/jenkins/pull/9453/commits/3c676769eb00bddc8d6f6af59960f32034286346.

I was unable to identify a difference in behavior as a result of this PR (e.g., in the "Copy From" textfield on the new item dialog), so perhaps I'm just missing how this is supposed to work? If it's not needed anymore or compensated for somehow by this new code, the `data-` attribute should not longer be set.

Pinging @zbynek as the original author who might be able to help with this question.

Reply:
My PR disables Tippy's default `hideOnClick` only for autocomplete, this PR disables it for all other dropdowns. So PR won't change behavior of autocomplete. It'ts the dropdowns that are *not* autocomplete that changed behavior in this PR: 

For context menus clicking the chevron (downward triangle) the second time no longer closes the context menu. If that's still a desired behavior, the `data-` attribute is still needed (and should be read in the `click` handler to decide how to handle clicks in `reference` element).

Reply:
@zbynek Thanks for clarifying. That seems like a problem.

https://github.com/user-attachments/assets/fb39fc7a-9ea2-4760-ad15-c57776a02ed2

Reply:
@daniel-beck I have updated the code to fix this problem. Thank you.

https://github.com/user-attachments/assets/87fb6dad-07dc-4947-af82-32811572dc3a

Reply:
@ridemountainpig Thanks. Unfortunately now the fields where this was previously left open close when they shouldn't:

https://github.com/user-attachments/assets/c36959ec-9700-46dc-af24-16765f948f8f

That's what the `hideOnClick` was used for. Could this behavior be retained, or does the fix for this require that we change one of the previous behaviors (stay open on autocomplete / close in other cases)?

Reply:
@daniel-beck I have updated it. Could you please check it again? Thank you.

https://github.com/user-attachments/assets/843cf75b-514e-4de0-9798-4057bff0e72f